system/models/users/register.py

```diff
@@ -0,0 +1,15 @@
+import hashlib
+import os
+
+class Register:
+    def __init__(self, user_id: int, user_name: str, password: str, is_admin: bool, error: str):
+        self._user_id = user_id
+        self._user_name = user_name
+        self._password = password
+        self._is_admin = is_admin
+
+    def ask_user_credentials(self):
```

Comment:
Make sure to type cast the return value with "def ask_user_credentials(self): -> return value type"
Although this implementation looks like its only temporary so thats fine for now.